src/test/util/mod.rs

```diff
@@ -172,6 +172,15 @@ impl TestClient {
         coll
     }
 
+    pub async fn init_db_and_typed_coll<T>(&self, db_name: &str, coll_name: &str) -> Collection<T>
+    where
+        T: Serialize,
+    {
+        let coll: Collection<T> = self.database(db_name).collection_with_type(coll_name);
```

Comment:
Interesting, does the compiler require the type annotation on `coll` here?

Reply:
oh that must've been a leftover from a previous implementation, it's not necessary